src/renderer/store/actions.js

```diff
@@ -150,13 +150,13 @@ async function fetchCommitState({ commit }) {
   }
 }
 
-async function fetchHoldings({ commit }, { done }) {
+async function fetchHoldings({ commit }, { done, isRequestSilent }) {
   const currentNetwork = network.getSelectedNetwork();
   const currentWallet = wallets.getCurrentWallet();
   let portfolio;
   let holdings;
 
-  commit('startRequest', { identifier: 'fetchHoldings' });
+  commit('startRequest', { identifier: 'fetchHoldings', isSilent: isRequestSilent });
```

Comment:
Maybe abstract this into something like `startSilentRequest` instead of requiring another option.

Reply:
Done